sycl/doc/extensions/experimental/sycl_ext_oneapi_bfloat16_math_functions.asciidoc

```diff
@@ -184,7 +187,195 @@ T fabs(T x);
 
 ===== Description
 
-Compute absolute value of a `bfloat16`.
+Compute absolute value of a `bfloat16` value or `sycl::marray<bfloat16, N>`.
+
+==== ceil
+
+```c++
+namespace sycl::ext::oneapi::experimental {
+template <typename T>
+T ceil(T x);
+} // namespace sycl::ext::oneapi::experimental
+```
+
+===== Description
+
+Round to integral value using the round to positive infinity rounding mode
+for a `bfloat16` value or `sycl::marray<bfloat16, N>`.
```

Comment:
Nit: we already have stated above:
"The following functions are only available when T is bfloat16 or sycl::marray<bfloat16, {N}>, where {N} means any positive value of size_t type."
```suggestion
```